tasks/app/__init__.py

```diff
@@ -5,40 +5,40 @@
 
 from invoke import Collection
 
-from . import (
```

Comment:
Is there some logic about the order that the imports and the Collection is created? 
If there is, a comment would help to prevent future errors.

Reply:
No, the order here does not matter.  What matters is that the modules we import do not import objects in app/models/ out of order.  The way to ensure this is to move all module-level imports to function-level imports.  We want the order of the modules to be defined in `config.py` because we have database ORM dependencies.  For example if we import `app.modules.assets.models.Asset` for a test, it may be that it is imported before the `submission` module, breaking the import structure we have setup.  The order of these imports shouldn't matter.